### PR TITLE
Revert "pow pow pow"

### PR DESCRIPTION
**Pull request by casche**
Reverts reviewninja/foo#14

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://staging.review.ninja/reviewninja/foo/pull/15" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/15" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/15" target="_blank">ReviewNinja

File: math.js [hunk 1]
```diff
@@ -14,9 +14,5 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
-  },
-  
-  pow: function(a, b) {
-    return a ^ b;
```
Comment (casche) [inline math.js]: !fix you need to fix this code

Comment (casche) [inline math.js]: !fixed

File: math.js [hunk 2]
```diff
@@ -14,9 +14,5 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
-  },
```
Comment (casche) [inline math.js]: !fix you need to remove this whitespace

Comment (casche) [inline math.js]: !fixed

File: math.js (hunk 1 above)
Comment (casche) [inline math.js]: !resolve

Comment (casche) [inline math.js]: !fixed

**Comment by dfarr** [comment]
⭐

File: math.js [hunk 3]
```diff
@@ -14,9 +14,5 @@ module.exports = {
...
-  
-  pow: function(a, b) {
```
Comment (dfarr) [inline math.js]: !fix ix this code

Comment (dfarr) [inline math.js]: !resolved

File: math.js [hunk 4]
```diff
@@ -14,9 +14,5 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
-  },
-
```
Comment (dfarr) [inline math.js]: !fix you're missing a semi colon

Comment (dfarr) [inline math.js]: !fixed

File: math.js (hunk 2 above)
Comment (dfarr) [inline math.js]: `test`

Comment (dfarr) [inline math.js]: `test this` vs `something else`

Comment (dfarr) [inline math.js]: `one more that is longer and some more stuff yah....`

Comment (dfarr) [inline math.js]: `asfdsdfa_asfdfsa` asdf`

**Comment by dfarr** [comment]
lgtm

**Comment by dfarr** [comment]
LGTM

**Comment by dfarr** [comment]
lgtmadsafs

**Comment by dfarr** [comment]
asddfLGTM

File: math.js [hunk 5]
```diff
@@ -14,9 +14,5 @@ module.exports = {
   
   multiply: function(a, b) {
```
Comment (dfarr) [inline math.js]: `render this`

**Comment by dfarr** [comment]
test

File: math.js (hunk 3 above)
Comment (dfarr) [inline math.js]: !fixed

File: math.js (hunk 1 above)
Comment (dfarr) [inline math.js]: !fixed

File: math.js (hunk 5 above)
Comment (dfarr) [inline math.js]: adding a comment

Comment (dfarr) [inline math.js]: !fix

Comment (dfarr) [inline math.js]: !resolved

Comment (dfarr) [inline math.js]: dude

**Comment by dfarr** [comment]
hello world

**Pull request merged**